### PR TITLE
Dev-6039 Mobile Dismiss Message

### PR DESCRIPTION
**Pull request by jonhill13**
**High level description:**

High level description of the PR.

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-6039](https://federal-spending-transparency.atlassian.net/browse/DEV-6039)

**Mockup:**
https://bahdigital.invisionapp.com/share/WDIAEVZ2B7U#/screens/296048764

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [N/A] Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
- [N/A] All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)
- [N/A] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [N/A] Design review complete `if applicable`
- [N/A] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/_scss/mobileMessage.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.mobile-message__container {
+    position: fixed;
+    bottom: 9px;
+    background-color: $color-gray-dark;
+    color: $color-white;
+    z-index: 9999999999999999999;
```
Comment (MaxwellKendall) [inline src/_scss/mobileMessage.scss:6]: smh

File: src/_scss/mobileMessage.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      }
+      .mobile-message__text {
+        font-size: 1.4rem;
+        letter-spacing: 0;
+        line-height: 1.8rem;
+        padding-right: 3.9rem;
+      }
+    }
+    .mobile-message__button {
+      float: right;
```
Comment (MaxwellKendall) [inline src/_scss/mobileMessage.scss:29]: wut.. float? what is this the 90s?

File: src/js/components/sharedComponents/MobileMessage.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            setHideMessage('hide');
+        }
+    };
+    // remove disclaimer after 5 seconds
+    useEffect(() => {
+        if (Cookies.get(cookie) !== 'hide') {
+            setTimeout(() => {
+                Cookies.set(cookie, { expires: 1 });
+                setHideMessage('hide');
+            }, 10000);
```
Comment (MaxwellKendall) [inline src/js/components/sharedComponents/MobileMessage.jsx:34]: isn't this 10 seconds?

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
Looking at this in the browser doesn't look good.

File: src/js/components/sharedComponents/MobileMessage.jsx (hunk 3 above)
Comment (jonhill13) [reply src/js/components/sharedComponents/MobileMessage.jsx:34]: yes

Comment (MaxwellKendall) [reply src/js/components/sharedComponents/MobileMessage.jsx:34]: The comment says 5

File: src/_scss/mobileMessage.scss (hunk 1 above)
Comment (jonhill13) [reply src/_scss/mobileMessage.scss:6]: please provide useful PR comments

File: src/js/components/sharedComponents/MobileMessage.jsx (hunk 3 above)
Comment (jonhill13) [reply src/js/components/sharedComponents/MobileMessage.jsx:34]: yeah, design review we made it 10

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**